src/server/serverBase.ts

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License
  **********************************************************/
+// this file is the legacy controller for local development, until we move server side code to use electron's IPC pattern and enable electron hot reloading
```

Comment:
as the comment mentioned, this file is legacy, so there are some duplicate code if compare this file to EventHubHandler.ts
We haven't found a good way to enable electron local debugging during the RBAC refactor, so it is currently using serverBase in local development mode, while in electron mode using EventHubHandler
A bit more info in wiki: https://github.com/Azure/azure-iot-explorer/wiki/Dev-manual#how-to-debug-electron-main-dev